app/views/sites/usage_rules/edit.html.slim

```diff
@@ -1,60 +1,81 @@
+    = semantic_form_for @settings, url: admin_site_usage_rules_path, html: { class: "pf-c-form pf-m-limit-width" } do |form|
+      section class="pf-c-form__section" role="group" aria-labelledby="settings-section-user-account"
+        div class="pf-c-form__section-title" id="settings-section-user-account" aria-hidden="true" User account management zone
+        div class="pf-c-form__group"
+          div class="pf-c-form__group-control"
+            = render 'settings_check', setting: 'useraccountarea_enabled', checked: settings.useraccountarea_enabled
 
-    - if can?(:see, :account_plans) || can?(:see, :service_plans)
-      = form.inputs 'Advanced Plans'
-        - if can? :see, :account_plans
-          = form.input :account_plans_ui_visible, input_html: { class: 'toggle-plan', data: { targets: 'use-account-plans'}}
+      - if current_account.master? && !ThreeScale.config.onpremises
+        section class="pf-c-form__section" role="group" aria-labelledby="settings-section-hide-service"
+          div class="pf-c-form__section-title" id="settings-section-hide-service" aria-hidden="true" Hide services
+          div class="pf-c-form__group"
+            div class="pf-c-form__group-control"
+              = render 'settings_check', setting: 'hide_service', checked: settings.hide_service
 
-          = form.inputs "Account Plans Changing", class: "use-account-plans #{hide_class_unless_account_plans_visible(@settings)}" do
-            = render 'shared/plan_change_settings',  form: form, permission: :change_account_plan_permission, label: ""
+      section class="pf-c-form__section" role="group" aria-labelledby="settings-section-signup"
+        div class="pf-c-form__section-title" id="settings-section-signup" aria-hidden="true" Signup
+        div class="pf-c-form__group"
+          div class="pf-c-form__group-control"
+            = render 'settings_check', setting: 'signups_enabled', checked: settings.signups_enabled
+        - if @settings.authentication_strategy.cas?
+          div class="pf-c-form__group"
+            div class="pf-c-form__group-control"
+              = render 'settings_check', setting: 'cas_server_url', checked: settings.cas_server_url != '0'
+        - if @settings.approval_required_editable?
+          div class="pf-c-form__group"
+            div class="pf-c-form__group-control"
+              = render 'settings_check', setting: 'account_approval_required',
+                                         checked: settings.account_approval_required,
+                                         hint: approval_required_editable_hint(current_account)
+        - elsif @settings.approval_required_disabled?
+          div class="pf-c-form__group"
+            div class="pf-c-form__group-control"
+              = render 'settings_check', setting: 'account_approval_required',
+                                         checked: settings.account_approval_required,
+                                         disabled: true,
```

Comment:
missing handing of the `disabled` property in the `settings_check` partial?...

Reply:
Oops, I forgot